packages/blockchain-extension/test/commands/gatewayConnectCommand.test.ts

```diff
@@ -73,7 +71,7 @@ describe('GatewayConnectCommand', () => {
         let choseIdentityQuickPick: sinon.SinonStub;
         let chosenGatewayQuickPick: sinon.SinonStub;
         let chosenWalletQuickPick: sinon.SinonStub;
-        let identity: IdentityInfo;
+        let identity: any;
```

Comment:
Was there a reason for changing this?

Reply:
as otherwise you need to import `fabric-client` and we need to not do that in the extension package